rules/cross-platform/execution_revershell_via_shell_cmd.toml

```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2020/01/07"
+maturity = "production"
+updated_date = "2020/01/07"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies the execution of a shell process with suspicious arguments which may be indicative of reverse shell activity."
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Potential Reverse Shell Activity via Terminal"
+references = [
+    """
+    https://github.com/swisskyrepo/PayloadsAllTheThings/blob/master/Methodology and Resources/Reverse Shell
+    Cheatsheet.md
+    """,
```

Comment:
was this linted this way over 2 lines? IINM it should keep links in tact, even if exceeded max line width. Also, can you url encode the link with `%20`

Reply:
encoded.